code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/identity/TokenResourceManager.java

```diff
@@ -115,4 +138,30 @@ public void setTokenCreators(List<TokenCreator> tokenCreators) {
         this.tokenCreators = tokenCreators;
     }
 
+    @Override
+    protected Object deleteInternal(String type, String id, Object obj, ApiRequest request) {
+        if (!StringUtils.equals(AbstractTokenUtil.TOKEN, request.getType())) {
+            return null;
+        }
+        return deleteToken(obj, request);
```

Comment:
Was just talking to @vincent99 about this check (which is in the below deleteToken method) and we want to change this. We actually don't want the UI to have to know and remember that ID. The cookie with the token should be enough info.

If you need to have an ID to work around cattle API quirks regarding DELETE on collections, then we can make the ID some constant string like "current" (ie /v2-beta/tokens/current)...this could also set us up in the future for implementing a /v2-beta/tokens/all  to invalidate all tokens for the current user.